helm-mapkubeapis.yaml

```diff
@@ -0,0 +1,104 @@
+package:
+  name: helm-mapkubeapis
+  version: 0.6.1
+  epoch: 0
+  description: Helm plugin to map and update deprecated Kubernetes APIs in Helm release manifests
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/helm/helm-mapkubeapis
+      tag: v${{package.version}}
+      expected-commit: afa30453b58395ad8215cfe2c1a1edff50428baa
+
+  - uses: go/bump
+    with:
+      deps: |-
+        github.com/docker/docker@v26.0.0-rc3
+        golang.org/x/oauth2@v0.27.0
+        helm.sh/helm/v3@v3.17.4
+
+  # Dropping helm plugins in /usr/libexec since they are binaries and
+  # configuration intended to be run by the helm command.
+  # See https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s07.html
+  # Installers of this package will have to set this environment var
+  # HELM_PLUGINS=/usr/libexec/helm-plugins
+  - runs: |
+      cat Makefile
+      make build
+      INSTALL_DIR="${{targets.destdir}}/usr/libexec/helm-plugins/${{package.name}}"
+      mkdir -p "${INSTALL_DIR}"
+      mkdir -p "${INSTALL_DIR}/bin"
+      mkdir -p "${INSTALL_DIR}/config"
+
+      install -Dm755 ./bin/mapkubeapis "${INSTALL_DIR}/bin/mapkubeapis"
+      install -Dm644 plugin.yaml "${INSTALL_DIR}/plugin.yaml"
+      install -Dm644 config/Map.yaml "${INSTALL_DIR}/config/Map.yaml"
+
+update:
+  enabled: true
+  github:
+    identifier: helm/helm-mapkubeapis
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - helm
+    environment:
+      KUBERNETES_SERVICE_HOST: "127.0.0.1"
+      KUBERNETES_SERVICE_PORT: 32764
+      HELM_PLUGINS: "/usr/libexec/helm-plugins"
+  pipeline:
+    - runs: |
+        helm env
+        helm plugin list 2>&1 | grep -i 'mapkubeapis'
+    - uses: test/kwok/cluster
+    - name: "Tests for helm-mapkubeapis plugin"
+      runs: |
+        set -x
+        mkdir -p /tmp/deprecated-chart/templates
+        export KUBECONFIG=$HOME/.kwok/clusters/kwok/kubeconfig.yaml
+        cat > /tmp/deprecated-chart/Chart.yaml <<EOF
+        apiVersion: v2
+        name: deprecated-chart
+        version: 0.1.0
```

Comment:
Where are we getting this version from?

Reply:
the version is just a placeholder for test chart we dynamically create during the test run. It's not sourced from anywhere just a simple valid version field required by helm chart spec for it to install correctly.